autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectColumn.css

```diff
@@ -34,6 +34,10 @@
     padding-top: 8px;
     padding-right: 8px;
     box-sizing: border-box;
+    min-height: auto;

```

Comment:
I actually don't see any scroll bar on my computer, on either Chrome or Firefox. Can you confirm that the scroll bar still shows up on yours?

Reply:
This is fixed with your most recent changes

Reply:
Checked this on my own screen again, and using `38px` doesn't work on my screen if the course title is really long (see below). Do you think we could just remove the course title? I think that if we only showed the department and course number, it would always fit on 1 line no matter the screen size.

![image](https://user-images.githubusercontent.com/10082177/103492870-a07efa80-4df3-11eb-8bae-e8e8af5a6be8.png)